src/Sulu/Bundle/ContentBundle/Resources/views/Template/content.html.twig

```diff
@@ -11,6 +11,7 @@
             {% set type = get_type(property.contentTypeName) %}
 
             {% if property.contentTypeName == 'block' %}
+                {% include 'SuluContentBundle:Template:macros/single.html.twig' with {'property': property, 'params': params, 'type': type, 'id': property.name, 'webspaceKey': webspaceKey, 'languageCode': languageCode, 'userLocale': userLocale} only %}
                 {% include type.template with {'property': property, 'params': params, 'type': type, 'id': property.name, 'webspaceKey': webspaceKey, 'languageCode': languageCode, 'userLocale': userLocale} only %}
```

Comment:
@peric for me it destroys the form when i add this line ... we should add the title in the template `SuluContentBundle:Template:content-types/block.html.twig` and adapt the css. @berndhep where should the title be place above "all" block once or over every single block? /cc @danrot